setup.py

```diff
@@ -6,17 +6,26 @@
 ]
 
 # setup.
-setup(name='dl-utils',
-      version='0.2.0',
-      description='deep learning utilities',
-      author=', '.join(contrib),
-      license='MIT',
-      packages=find_packages(exclude=[
-          'tests',
-      ]),
-      install_requires=[
-          'numpy', 'scipy', 'tensorflow>=2.3,<2.4', 'scikit-image>=0.13',
-          'scikit-learn', 'future', 'tqdm', 'pytest', 'pyyaml', 'h5py',
-          'tensorflow-addons==0.11.2'
-      ],
-      zip_safe=False)
+setup(
+    name='dl-utils',
+    version='0.2.0',
+    description='deep learning utilities',
+    author=', '.join(contrib),
+    license='MIT',
+    packages=find_packages(exclude=[
+        'tests',
+    ]),
+    install_requires=[
+        'numpy<=1.19.5',  # numpy 1.2 incomptabile with tensorflow 2.3, see https://stackoverflow.com/questions/66207609/notimplementederror-cannot-convert-a-symbolic-tensor-lstm-2-strided-slice0-t
```

Comment:
Does upgrading to tensorflow-2.4 mitigate this issue?

Reply:
I'm not entirely sure, this is quite recent. Anyway other errors come up (e.g. failed model import/export test).

This can all be fixed at once in a few months and once things have stabilized. For the model import/export, it's typically an issue with the architecture export to yaml.